docs/language/data-types.md

```diff
@@ -108,7 +108,7 @@ type socket = {addr:ip,port:port=uint16}
 defines a named type `socket` that is a record with field `addr` of type `ip`
 and field `port` of type "port", where type "port" is a named type for type `uint16` .
 
-Named types may also be defined by the input data itself, as super data is
+Named types may also be defined by the input data itself, as super-structured data is
```

Comment:
I'm making all attempts to avoid the use of "super data" as a standalone adjective+noun pairing. On a case by case basis, I've been trying to replace with "super-structured data" or some reference to the "super data model".

Reply:
I thought we were going with polymorphic instead of super-structured.

Reply:
I think polymorphic is good when explaining the theoretical foundations when talking to a more academic audience but maybe we should avoid it as the headline.  Super-structured is good when comparing/contrasting with semi-structured and structured data.  "Super data model" is good for the data model and "super data" I'm actually ok with as a generic terms to mean any data that conforms to the super data model, e.g., there is super data inside of a CSUP file,  the super.Value Go type represents super data, super data is a superset of relational and JSON data.  So I guess here I would just leave it as "super data" but super-structured is ok too.

Reply:
👍